extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.tracking.js

```diff
@@ -1,6 +1,8 @@
 define('wikia.articleVideo.featuredVideo.tracking', ['wikia.tracker'], function (tracker) {
 	var state = getDefaultState(),
 		defaultGACategory = 'featured-video',
+		//This will replace 'trackingevent' in internal tracker url path
+		eventName = 'videoplayerevent',
```

Comment:
maybe we should store such things in different "constants" module / config module?

Reply:
We could, but we use these values literally - storing them separately would make sense if we'd use the entire config objects somewhere but we just use the 2 variables in the tracker for now, so this seems easier and more readable for me. What do you think?